CHANGELOG.rst

```diff
@@ -7,6 +7,8 @@ CHANGELOG
 
 * doc-fix: update information about saving models in the MXNet README
 * doc-fix: change ReadTheDocs links from latest to stable
+* feature: Support for predictor to delete endpoint and endpoint configuration
```

Comment:
You also added support to delete model through the Model class.

I would also call out that we by default delete the endpoint configuration in ``delete_endpoint``